linux/setup/test_setup.yml

```diff
@@ -9,7 +9,10 @@
   include_tasks: ../../common/set_current_testcase_facts.yml
 
 - name: "Check base snapshot existence and/or revert to it"
-  include_tasks: base_snapshot_check_revert.yml
+  include_tasks: ../../common/base_snapshot_check_revert.yml
+
+- name: "Check VM settings"
+  include_tasks: ../../common/check_vm_settings.yml
```

Comment:
I think we can add a variable at the end of `check_vm_settings.yml` to record the vm settings check is passed, and in `test_setup.yml`, only when the variable is undefined, it is needed to execute `check_vm_settings.yml`.  So that, only the first test case need to check VM settings, and the rest ones can skip it.

Reply:
Done.